apps/web/src/components/Table/ResponsiveTable/ColumnDialog.tsx

```diff
@@ -61,7 +66,7 @@ const ColumnDialog = <T extends object>({ table }: ColumnDialogProps<T>) => {
                     }}
                   />{" "}
                   {intl.formatMessage(adminMessages.toggleAll)}
-                </label>
+                </Field.Label>
               </div>
             </Field.BoundingBox>
```

Comment:
Something I noticed is that the bounding box only encompasses the first checkbox. @substrae Should we extend it to wrap all of the checkboxes?
![Screenshot 2023-09-28 105318](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/b8374856-b18b-469b-88e7-6bec20bd98e3)